src/app/clusters/commodity-tariff-server/app_config_dependent_sources.gni

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+app_config_dependent_sources = [
+  "CommodityTariffAttrsDataMgmt.cpp",
+  "CommodityTariffAttrsDataMgmt.h",
```

Comment:
Are these actually app config dependent?  Is the cluster server impl app config dependent?